### PR TITLE
Migrate to biny

### PR DESCRIPTION
**Pull request by matthewperiut**
Previously was using barn, since biny is a fork of barn it wasn't too intrusive.

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -14,7 +14,7 @@ maven_group = pl.telvarost
 archives_base_name = mojangfixstationapi
 
 # Dependencies
-stapi_version=2.0-alpha.2
+stapi_version=2.0-alpha.2.3
```
Comment (telvarost) [inline gradle.properties:17]: Isn't StationAPI 2.3 broken in prod right now? Shouldn't this be 2.1?

Comment (matthewperiut) [reply gradle.properties:17]: It doesn't make a difference, as the things changed in 2.3 versus 2.1 are not touched by this mod, so pretty much the only difference is the runtime version of stationapi as all API things remain the same.

Comment (matthewperiut) [reply gradle.properties:17]: and in dev it works fine

Comment (telvarost) [reply gradle.properties:17]: I usually like to set it to whatever, players will actually experience. So I will probably revert it back to 2.1 until StationAPI stabilizes

Comment (matthewperiut) [reply gradle.properties:17]: Sure, it doesn't make much of a difference to me

**Pull request merged**